### ISSUE TITLE
PI Node in a Virtual Machine

### ISSUE THREAD

**Issue by ghost** [pi-node/instructions#12]
It should be possible to allow to run natively a PI Node within a Virtual machine. But requiring to enable Hyper-V and docker, it limits the number of applicants, as it required Nested virtualization, that not every hypervisor support.

**Comment by serousoma** [comment]
Yes, the correct way will be native on linux for nodes, at some point. As with all other networks, linux users Will inevitably run most of the nodes.
